ppl/src/main/antlr/OpenSearchPPLLexer.g4

```diff
@@ -35,6 +35,7 @@ NEW_FIELD:                          'NEW_FIELD';
 KMEANS:                             'KMEANS';
 AD:                                 'AD';
 ML:                                 'ML';
+FILLNULL:                           'FILLNULL';
```

Comment:
add FILLNULL to the `keywordsCanBeId` [here](https://github.com/opensearch-project/sql/blob/566f8cdaeb8ba47b3b2cb28c6de0110b1b017696/ppl/src/main/antlr/OpenSearchPPLParser.g4#L845)

Reply:
@YANG-DB  I can add it, but I don't see it in the Spark code.
https://github.com/opensearch-project/opensearch-spark/blob/main/ppl-spark-integration/src/main/antlr4/OpenSearchPPLParser.g4#L1096

If this is needed, I can also put up a PR for Spark.